### DISCUSSION TITLE
Lesson 11: Metata Pinning

### DISCUSSION THREAD

**Discussion by alexkalen** [smartcontractkit/full-blockchain-solidity-course-py#1754] [Q&A] [answered]
Why do we only pin the images? Why dont we pin the metadata json file as well on Pinata?

**Comment by cromewar** [comment, marked as answer]
Hello @alexkalen 
You can do so, but we usually use those servers for heavy load (images) and the lightest ones we we the traditional way.

**Reply by alexkalen** [reply]
What would be the traditional way? Doesnt the json files need to be pinned in order to be accesed by opensea? @cromewar  thanks for the quick response

**Reply by cromewar** [reply]
when I said traditional I meant just using out local ifs node for that, as usually you will like to a market place which will pin that metadata such as opensea or a server like Moralis which runs the ipfs node for you. However you are right you can also upload the metadata to Piñata if you like so.
